client/components/date-range/shared-props.ts

```diff
@@ -0,0 +1,3 @@
+export default interface SharedProps {
+	isCompact: boolean;
+}
```

Comment:
The sharing is pretty minimal and this is very small, I'd consider inlining the prop where used, unless I misinterpreted something.

Reply:
Removed.